src/Jsonyte/Serialization/JsonMemberInfo.cs

```diff
@@ -181,6 +181,26 @@ public void WriteRelationship(Utf8JsonWriter writer, ref TrackedResources tracke
             }
         }
 
+        public void WriteRelationshipWrapped(Utf8JsonWriter writer, ref TrackedResources tracked, object resource)
```

Comment:
Kind of hate that this is becoming a dumping ground for 'routing' calls to different converters. Will need to revisit.